src/components/Navigation.js

```diff
@@ -57,26 +81,44 @@ function Navigation() {
     alertErrorRef.current.show(3000);
   };
 
-  return (
-    <div className="main-nav">
-      <nav className="navbar navbar-expand-lg navbar-dark ps-3 ps-lg-0">
-        <div className="d-flex flex-column align-items-center">
-          <Link className="navbar-brand" to="/" href="/">
-            <img src={logo} alt="" />
-          </Link>
+  const renderNewUi = () => {
+    return (
+      <nav>
+        <div className="hide-logo-container-mobile">
+          <div className="newLogo-explorer-container">
+            <div className="d-flex flex-column align-items-center">
+              <Link className="navbar-brand" to="/" href="/">
+                <NewLogo
+                  className={`newLogo ${theme === 'dark' ? 'dark-theme-logo' : 'light-theme-logo'}`}
+                />
+              </Link>
+            </div>
+            {!showSearchInput ? (
+              <button
+                className="navbar-toggler {}"
```

Comment:
question: Is the className `{}` necessary?

Reply:
removed.